libs/README.md

```diff
@@ -0,0 +1,28 @@
+# Libraries
```

Comment:
How is this folder different from `vehicle/software/mkx/libs`?

Reply:
Ahhh I see. This is where _shared_ libraries live, while board-specific libraries live in `vehicle/mkx/software/example_board/libs`. 

I think separating out share libraries is smart, but I'd still like to move this closer to the rest of the vehicle software; it feels weird to me to keep this at the root of the directory.

(I've also left a comment below about how we organize code by vehicle, check that out for more context).

Here's what I propose:

```
vehicle
|- hardware
|  |- all_the_hardware_stuff
|- software
   |- shared
   |  |- CAN
   |     |- can.c
   |     |- can.h
   |- example_board
      |- src
      |  |- example_board.c 
      |- libs
         |- all_board_specific_c_and_h_files
```

Reply:
But what about projects in the research phase that are using CAN? I don't think we should have cross dependencies between `projects/` and `vehicle/`. I think all _shared_ libraries should be in a common `libs/` folder. And something like `vehicle/software/example_board/libs/` would be a good candidate for something like the LTC BMS chip driver.

Reply:
that's a great point about research projects. it still irks me a little bit that libs sits at the root of the repo, but its definitely not as bad as I initially thought. I think I can be convinced, but I'd like to have a conversation about it.